models/amity.py

```diff
@@ -0,0 +1,374 @@
+import random
+from models.room import OfficeSpace
+from models.room import LivingSpace
+from models.person import Staff
+from models.person import Fellow
+
+class Amity(object):
+
+	def __init__(self):
+		self.all_people = []
+		self.staff_list = []
+		self.fellows_list = []
+		self.allocated_people = []
+		self.allocated_staff = []
+		self.allocated_fellows = []
+		self.unallocated_people = []
+		self.all_rooms = []
+		self.living_spaces = []
+		self.office_spaces = []
+		self.allocated_rooms = []
+		self.db_people_list = []
+		self.db_room_list = []
+
+	def create_room(self, room):
+		'''
+		First check that the new room being created does not 
+		already exist.
+		'''
+		for rooms in self.all_rooms:
+			if room["room_name"] == rooms.room_name:
+				print (rooms.room_name.upper()+" already exists.")
+				return
```

Comment:
Hey @njirap, reading up on list comprehensions, I've found that they must always return a list. Since this function is just a check to ensure rooms are not duplicated, I don't think I should use a list comprehension here since I'm just printing a string and exiting the function if the condition is true. Is that ok?